src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
you could try to use short form of this 
https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd